build/makelib/golang.mk

```diff
@@ -176,7 +176,7 @@ go.vet:
 
 .PHONY: go.fmt
 go.fmt:
-	@$(GOHOST) fmt $(GO_ALL_PACKAGES)
+	@gofmt_out=$$(gofmt -d -e $(GO_SUBDIRS) 2>&1) && [ -z "$${gofmt_out}" ] || (echo "$${gofmt_out}" 1>&2; exit 1)
```

Comment:
Using () creates a sub-shell, using {} does not.  This can be an issue in scripts, especially for "exit", as within () it will just exit from the sub-shell, but when using {} it'll exit from the script.